### DISCUSSION TITLE
How to set column width equal to header/title width?

### DISCUSSION THREAD

**Discussion by city17** [glideapps/glide-data-grid#439] [Q&A] [answered]
Hi, I've got a bunch of columns that are all checkboxes, but the titles are all different. Because each checkbox is small, the header title is the limiting factor for the width of the column.

Is it possible to set the width of each column equal to the minimal width of the column header? So that they all take up the least amount of space necessary, while ensuring the full title can be read?

If I leave width empty, it sets each column equal to the content (checkbox) width, which makes the titles unreadable.

<img width="355" alt="Schermafbeelding 2022-07-18 om 10 32 46" src="https://user-images.githubusercontent.com/13391662/179473627-e89e0cd9-3656-428b-9b22-0d8c96295696.png">

Alternatively if I set a specific width, some titles are readable, some not, and some take up a lot of white space. It would be great if I could set the width equal to the minimum size of the column header.

Thanks!

**Comment by jassmith** [comment, marked as answer]
This is already fixed in 5.0.0

**Reply by city17** [reply]
Cool, looking forward to 5.0.0 release. How will it be set there exactly? Or will this be the new default behavior?

**Reply by jassmith** [reply]
The width of the title is not intentionally ignored, it just is getting detected as an outlier in your use case. In 5.0 the title width is never detected as an outlier.
